src/hotspot/share/memory/arena.hpp

```diff
@@ -134,6 +134,15 @@ class Arena : public CHeapObj<mtNone> {
   void* Amalloc(size_t x, AllocFailType alloc_failmode = AllocFailStrategy::EXIT_OOM) {
     x = ARENA_ALIGN(x);  // note for 32 bits this should align _hwm as well.
     debug_only(if (UseMallocOnly) return malloc(x);)
+    // JDK-8270308: Amalloc guarantees 64-bit alignment and we need to ensure that in case the preceding
```

Comment:
Please ignore, this belongs to JDK-8270308 which is waiting for final approval (https://github.com/openjdk/jdk/pull/4835). Will be removed before pushing.